pandas/_version.py

```diff
@@ -205,10 +205,7 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, run_command=run_command):
         raise NotThisMethod("'git rev-parse' failed")
     full_out = full_out.strip()
 
-    pieces = {}
-    pieces["long"] = full_out
-    pieces["short"] = full_out[:7]  # maybe improved later
-    pieces["error"] = None
+    pieces = {"long": full_out, "short": full_out[:7], "error": None}
```

Comment:
I've did it. Is there a portion of the work that can be merged? I don't
mind closing it...

Le sam. 14 nov. 2015 20:00, Jeff Reback <notifications@github.com> a écrit :

> In pandas/_version.py
> <https://github.com/pydata/pandas/pull/11538#discussion_r44862081>:
>
> > @@ -205,10 +205,7 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, run_command=run_command):
> >          raise NotThisMethod("'git rev-parse' failed")
> >      full_out = full_out.strip()
> >
> > -    pieces = {}
> > -    pieces["long"] = full_out
> > -    pieces["short"] = full_out[:7]  # maybe improved later
> > -    pieces["error"] = None
> > +    pieces = {"long": full_out, "short": full_out[:7], "error": None}
>
> try
>
> git push sieben dict_litteral -f
>
> you can't push directly to origin, only to your fork
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/11538/files#r44862081>.
>